### ISSUE TITLE
Qusetion about pretrain for link prediction

### ISSUE THREAD

**Issue by jiaruHithub** [qiaoyu-tan/S2GAE#2]
For the link prediction problem, first unsupervised learning needs to be performed on the training set (e.g. 85% of the data in the Cora dataset is used for training). Which process is correct, please? (1) After unsupervised training with multiple epochs using unsupervised methods (e.g. S2GAE), train the predictor of the edge for prediction (2) Combine the loss of the self-supervised method with the loss of the edge predictor and train both parts simultaneously. (3) Direct training on the training set using an unsupervised method (containing the edge predictor). Since the edge predictor is included in the unsupervised method, the training is performed simultaneously. Afterwards, link prediction is used as an evaluation task only and no training is performed specifically for link prediction. For (3), the model obtained by the unsupervised method is actually selected based on the validation loss of the link prediction for the downstream task. (1) or (2) or (3)?

The second question is, can you tell me how it is done in the code?

Thank you very much!
